app/src/main/java/com/simprints/intentlauncher/data/db/IntentDatabase.kt

```diff
@@ -13,9 +13,10 @@ import androidx.room.RoomDatabase
     ],
     autoMigrations = [
         AutoMigration(from = 5, to = 6),
+        AutoMigration(from = 6, to = 7),
     ],
     exportSchema = true,
-    version = 6,
+    version = 8,
```

Comment:
Either this should be 7, or you did not add one migration. Also, I think that schema files should be checked into git as well.